tests/framework/ROM/TimeSeries/ARMA/segmented.xml

```diff
@@ -12,14 +12,14 @@
           In PR #882, the standard errors on statistics moments are implemented, these quantities are stored in
           the pointwise metadata, and will be printed out in the output csv files by default.
           In order to avoid regolding this test, the optional node 'what' with text values 'input, output' is added.
-          In this case, only the user requested variables as specified in the DataObjects are printed out. 
+          In this case, only the user requested variables as specified in the DataObjects are printed out.
       </revision>
```

Comment:
revision need to be updated since you have modified the test

Reply:
yes, added